python/composio/client/collections.py

```diff
@@ -516,7 +516,7 @@ def _handle_callback(
         try:
```

Comment:
Changing the logging level from `info` to `error` in `_handle_callback` is appropriate for capturing errors more prominently. This should help in better monitoring and debugging of callback failures.